backend/siarnaq/api/teams/filters.py

```diff
@@ -24,3 +24,10 @@ def transform(self, term):
         if term.startswith("-"):
             prefix, term = "-", term[1:]
         return prefix + self.FIELD_TRANSFORMS.get(term, term)
+
+
+class TeamActiveSubmissionFilter(filters.BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        if "has_active_submission" not in request.query_params:
+            return queryset
+        return queryset.filter(submissions__accepted=True).distinct()
```

Comment:
Little strange not to check true/false'ness of has active submission param in url